test/RoaveTest/SecurityAdvisories/VersionTest.php

```diff
@@ -107,38 +98,6 @@ public function testGreaterOrEqualThanVersionWith(
         self::assertSame($v2GreaterOrEqualThanV1, $version2->isGreaterOrEqualThan($version1));
     }
 
-    /**
-     * @dataProvider equivalentVersionProvider
-     *
-     * @param string $version1String
-     * @param string $version2String
-     */
-    public function testVersionEquivalence(string $version1String, string $version2String) : void
-    {
-        $version1 = Version::fromString($version1String);
```

Comment:
another case here. We do version normalization only during versions comparison and we rely on `version_compare()` there.